index.html

```diff
@@ -504,71 +475,72 @@ <h3>
           </p>
           <ul>
             <li>
-              <dfn>"navigation"</dfn>, if the request is a [=navigation
+              <code>"navigation"</code>, if the request is a [=navigation
               request=];
             </li>
             <li>
-              <dfn>"css"</dfn>, if the request is a result of processing a CSS
-              <a data-xref-type="css-function">url()</a> directive such as
+              <code>"css"</code>, if the request is a result of processing a CSS
+              <a data-cite="css-values-4"
```

Comment:
Is this not exported? If it isn't maybe file a bug against that repo?

Reply:
It's exported in both css-values-3 and css-values-4, which seems to create confusion in specref. Talking to @marcoscaceres, this is the syntax we want here..